notary-iroha-integration-test/src/main/kotlin/integration/helper/IrohaAccountHelper.kt

```diff
@@ -134,6 +134,16 @@ open class IrohaAccountHelper(private val irohaAPI: IrohaAPI, private val peers:
         createTesterAccount("consensus", listOf("consensus_collector"))
     }
 
+    /** Account that registers Ethereum addresses */
+    val ethRegistrationAccount by lazy {
```

Comment:
should it be used in registration tests in corresponding sidechain modules as well?

Reply:
I'm not sure about that.